label_studio/core/utils/common.py

```diff
@@ -744,3 +744,8 @@ def wrapper(*args, **kwargs):
 
 def empty(*args, **kwargs):
     pass
+
+
+def get_ttl_hash(seconds=60):
```

Comment:
Please add types for both `seconds` and the return value

Reply:
Added.